src/content/developers/tutorials/monitoring-geth-with-influxdb-and-grafana/index.md

```diff
@@ -0,0 +1,136 @@
+description: 
+author: "Mario Havel"
+tags: ["clients", "geth", "nodes"]
+skill: intermediate
+lang: en
+sidebar: true
+---
+
+Monitoring your node is a best practice which can help you to be informed about its performance, status, identify potential issues and even alert you if some problem occurs. Client implementations including [Geth](https://geth.ethereum.org/) offer thorough metrics collection and reporting. 
+
+## Prerequisites {#prerequisites}
+
+* Tutorial assumes that you are already running an instance of Geth client. 
+* Most of the steps and examples are for linux environment, basic terminal knowledge will be helpful. 
+* To get familiar with Geth's suite of metrics, its use for debugging and why it might be important to you, check talk [Monitoring an Ethereum infrastructure by Péter Szilágyi](https://www.youtube.com/watch?v=cOBab8IJMYI).
+
+
+## Monitoring stack {#monitoring-stack}
+
+Ethereum client collects various metrics from which a chronological database can be created. This database will be then read by data visualizing software. This means we need a stack of different tools working together. There are multiple variations of monitoring stack available with software like Prometheus (pull model), InfluxDB (push model), Telegraf, Grafana, Datadog, Chronograf, etc. 
+ 
+Preconfigured options worth mentioning are [Geth Prometheus Exporter](https://github.com/hunterlong/gethexporter) which enables easy setup using docker and [Ethbian](https://ethbian.org/index.html) OS for RPi 4 which comes with InfluxDB and Grafana preconfigured. 
+
+In this tutorial, we will manually setup InfluxDB with Grafana. Expected outcome is Geth pushing metrics data to InfluxDB which creates time series database and Grafana dashboard using this database as a source for graph visualization. Manual setup should teach you to understand the process, alter it and deploy on different environments. 
+
+## Setting up InfluxDB {#setting-up-influxdb}
+
+First, let's download and install InfluxDB. Various download options can be found at [Influxdata release page](https://portal.influxdata.com/downloads/), pick the one suiting your environment. 
+You can also install it from a [repository](https://repos.influxdata.com/). For example in Debian based distribution: 
+```
+curl -tlsv1.3 --proto =https -sL https://repos.influxdata.com/influxdb.key | sudo apt-key add 
+source /etc/lsb-release
+echo "deb https://repos.influxdata.com/${DISTRIB_ID,,} ${DISTRIB_CODENAME} stable" | sudo tee /etc/apt/sources.list.d/influxdb.list
+sudo apt update
+sudo apt install influxdb -y
+sudo systemctl enable influxdb
+sudo systemctl start influxdb
+sudo apt install influxdb-client
+```
+
+After successfully installing InfluxDB, make sure it's running on background. By default, it is reachable at `localhost:8086`. 
+Before using `influx` client, you have to create new user with admin privileges. This user will serve for high level management, creating databases and users. 
```

Comment:
Before using `influx` client, you have to create a new user with admin privileges. This user will be able to create databases and users.